README.md

```diff
@@ -509,6 +509,17 @@ You can use `g.setColor(r,g,b)` OR `g.setColor(16bitnumber)` - some common 16 bi
 
 [Graphics Class](https://banglejs.com/reference#Graphics)
 
+## Code Formatting
+
+BangleApps uses [Prettier](https://prettier.io/) for code formatting, with default settings.
+
+You can use it [online](https://prettier.io/playground.html) or on your computer:
+
+1. Install Prettier with `npm install prettier --global`.
```

Comment:
I wouldn't use a globally installed prettier version, since this forces everyone to install exactly the same version.

Instead, I would add a particular version of prettier as `devDependency` to `package.json`

```json
  "devDependencies": {
    "prettier": "2.3.0"
  },
```

Then you can run `npm install` or `npm ci` to install a local copy of prettier.

Reply:
npm scripts in `package.json` can then be used to invoke prettier.

```json
  "scripts": {
    "check-format": "prettier --check .",
    "fix-format": "prettier --write .",
  },
```

`npm run check-format`

`npm run fix-format`

Reply:
```suggestion
1. Install Prettier and other dependencies with `npm install` while in BangleApps.
```